core-bundle/composer.json

```diff
@@ -86,6 +86,7 @@
         "nikic/php-parser": "^4.9",
         "nyholm/psr7": "^1.2",
         "paragonie/constant_time_encoding": "^2.2",
+        "paragonie/csp-builder": "dev-master",
```

Comment:
The following 2 PRs have not been merged and/or released yet, thus I am (temporarily) requiring `dev-master` here:

* https://github.com/paragonie/csp-builder/pull/65
* https://github.com/paragonie/csp-builder/pull/74

Execute `composer require paragonie/csp-builder:dev-master` in your setup in order to test this PR (without having to lower the minimum stability).